docs/connect.html

```diff
@@ -320,6 +320,7 @@ <h4><a id="connect_rest" href="#connect_rest">REST API</a></h4>
 
     <ul>
         <li><code>GET /connector-plugins</code>- return a list of connector plugins installed in the Kafka Connect cluster. Note that the API only checks for connectors on the worker that handles the request, which means you may see inconsistent results, especially during a rolling upgrade if you add new connector jars</li>
+        <li><code>GET /connector-plugins/{connector-type}/config</code> - get the configuration definition for the specified connector plugin.</li>
```

Comment:
This endpoint works on all plugin types, not just connectors, right? Should we be try to be clearer about that?

```suggestion
        <li><code>GET /connector-plugins/{plugin-type}/config</code> - get the configuration definition for the specified plugin.</li>
```